packages/permissions/src/components/restricted-by-permissions/restricted-by-permissions.spec.tsx

```diff
@@ -3,29 +3,42 @@ import { ApplicationContextProvider } from '@commercetools-frontend/application-
 import { render } from '@testing-library/react';
 import RestrictedByPermissions from './restricted-by-permissions';
 
-type TPermissions = {
-  [key: string]: boolean;
+type TAllAppliedPermission = {
+  name: string;
+  value: boolean;
 };
-type TActionRight = {
-  [key: string]: boolean;
+type TAllAppliedActionRight = {
+  name: string;
+  value: boolean;
+  group: string;
 };
-type TActionRights = {
-  [key: string]: TActionRight;
+type TAllAppliedMenuVisibility = {
+  name: string;
+  value: boolean;
+};
+type TAllAppliedDataFence = {
+  __typename: 'StoreDataFence';
```

Comment:
Hm. @adnasa shouldn't this be not here?

Reply:
The generated types marked this as required, I assume because of the union type expansion

https://github.com/commercetools/merchant-center-application-kit/blob/bd3eb0d9b2817c1bab08a82e3f558c2853167223/packages/application-shell-connectors/src/types/generated/mc.ts#L824